src/sentry/workflow_engine/models/detector.py

```diff
@@ -63,14 +59,6 @@ class Detector(DefaultFieldsModel, OwnerModel, JSONConfigBase):
     # The user that created the detector
     created_by_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
 
-    class Meta(OwnerModel.Meta):
-        constraints = OwnerModel.Meta.constraints + [
-            UniqueConstraint(
-                fields=["organization", "name"],
-                name="workflow_engine_detector_org_name",
-            )
-        ]
-
```

Comment:
Should this be on the project level now? Or do we not care about uniqueness of names?

Reply:
good question, let me ask josh

Reply:
ok yep updated the constraint to project